test/TaxonAPI_server_test.py

```diff
@@ -169,39 +172,55 @@ def test_get_aliases(self):
 
     def test_get_info(self):
         ret = self.getImpl().get_info(self.getContext(), self.taxon)
+        ws_id, obj_id, version = self.wsClient.get_objects2({'objects': [{'ref': self.taxon}]})['data'][0]['path'][0].split('/')
+        # data saved on ci and appdev have different save date
+        del ret[0]['save_date']
+        # 'object_checksum' includes the date the object is saved, this info on ci and appdev are different
+        # needs to be compared particularly
+        if 'ci' in self.wsURL:
```

Comment:
I think this is fine for now but maybe we should just omit this test.  I'm not sure if it is really adding much value.